addons/website_event_meet/models/event_event.py

```diff
@@ -50,3 +50,11 @@ def _compute_meeting_room_allow_creation(self):
                 event.meeting_room_allow_creation = True
             elif not event.community_menu or not event.meeting_room_allow_creation:
                 event.meeting_room_allow_creation = False
+
+    def _get_website_menu_entries(self):
+        self.ensure_one()
+        original_entries = super()._get_website_menu_entries()
+        filtered_entries = [entry for entry in original_entries if entry[-1] != 'community']
```

Comment:
```suggestion
        filtered_entries = [entry for entry in super()._get_website_menu_entries() if entry[-1] != 'community']
```
had a question :thinking: 
in this case are we sure that the last menu is the community menu what if the menu is created by the user using the edit event menu?

Reply:
No, it should not cause any issue as call doesn't go to the the function `_get_website_menu_entries` when user creates new menu via frontend.